gatsby-config.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  siteMetadata: {
+    title: `ShelterTech - Technology for the underserved`,
+    description: `Blog example for Gatsby & Prismic`,
+    author: `Sheltertech`,
```

Comment:
```suggestion
    author: `ShelterTech`,
```

The "T" in Tech should be capitalized.